src/Whoops/Util/TemplateHelper.php

```diff
@@ -318,4 +322,24 @@ public function getCloner()
         }
         return $this->cloner;
     }
+
+    /**
+     * Set the application root path.
+     *
+     * @param string $applicationRootPath
+     */
+    public function setApplicationRootPath($applicationRootPath)
+    {
+        $this->applicationRootPath = $applicationRootPath;
+    }
+
+    /**
+     * Return the application root path.
+     *
+     * @return string
+     */
+    public function getApplicationRootPath()
```

Comment:
All setters in this class have getters, too. That's the reason why I've added the getter.